frontend/src/containers/Ledger.js

```diff
@@ -153,34 +153,41 @@ export class Ledger extends Component {
 
   }
 
+  loadCollectiveData() {
+    const { collective, params, fetchProfile } = this.props;
+    if (collective && collective.id) return Promise.resolve(collective);
+    return fetchProfile(params.slug).then(result => {
```

Comment:
I want to return something because I'm using `(collective)` in the `this.loadCollectiveData().then((collective))`.